src/main/groovy/edu/oregonstate/mist/locations/frontend/resources/LocationResource.groovy

```diff
@@ -26,7 +26,9 @@ import java.util.regex.Pattern
 class LocationResource extends Resource {
     private static final Logger LOGGER = LoggerFactory.getLogger(LocationResource.class)
 
-    public static final ArrayList<String> ALLOWED_CAMPUSES = ["corvallis", "extension"]
+    public static final ArrayList<String> ALLOWED_CAMPUSES = ["corvallis", "extension",
+                                                              "cascades","hmsc","other"]
```

Comment:
Could you put a space in between each campus? like: "cascades", "hmsc", "other]

Reply:
Fixed.